include/koa/authentication/passport-strategies/salesforce_authentication_strategy.js

```diff
@@ -0,0 +1,74 @@
+const CustomStrategy = require('passport-custom').Strategy;
+const strategyServices = require('./strategy_services');
+const request = require('request-promise');
+const Promise = require('bluebird');
+const SALESFORCE_OAUTH_TOKEN_URL = process.env.SALESFORCE_OAUTH_TOKEN_URL || '';
+const SALESFORCE_OAUTH_AUTHORIZE_URL = process.env.SALESFORCE_OAUTH_AUTHORIZE_URL || '';
```

Comment:
I am not sure if it matters at all.  But should be we accessing process.env.VARIABLE directly?  or should we be referencing pb.config.something and that something in the config should reference the process.env variable?

🤷‍♂️  (Super amused github makes the male shrug emoji out of the female shrug and a male gender symbol.  Random aside.)

Reply:
I took a look to the other files and we use the process.env, you can take a look at the BaseController